ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py

```diff
@@ -98,6 +98,9 @@ def remove_unchanged_or_null_args(server_resp, **kwargs):
             found_changed_list = False
             if len(value) == 0:
                 continue
+            if len(value) != len(server_value):
```

Comment:
if no of items in list differs.. then list is anyway not same. hence no need to check further